tests/cases/fourslash/autoImportNonModules.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: /tsconfig.json
+//// {
+////  "compilerOptions": {
+////    "module": "none",
+////  }
+////}
+
+// @Filename: /src/dirA/index.ts
+//// export * from "./thing1A";
+//// export * from "./thing2A";
+
+// @Filename: /src/dirA/thing1A.ts
+//// export class Thing1A {}
+//// Thing/**/
+
+// @Filename: /src/dirA/thing2A.ts
+//// export class Thing2A {}
+
+verify.completions({
+  marker: "",
+  excludes: ["Thing2A"],
+  preferences: {
+    includeCompletionsForModuleExports: false

```

Comment:
This has to be true for it to be testing your code. This is the equivalent of just turning auto-imports off in VS Code, so it has nothing to do with `module: none`

Reply:
done